src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/AppSecondFactor/index.tsx

```diff
@@ -34,9 +41,23 @@ export const AppSecondFactor: React.FC<AppSecondFactorProps> = ({
     false
   )
   const [stagedSecondFactor, setStagedSecondFactor] = useState(null)
-  const [isDisabling] = useState(false)
+  const [isDisabling] = useState(false) // ???
   const [isCreating, setCreating] = useState(false)
 
+  // Reset's _all_ of these state flags.
+  // NOTE: Don't do this. Just hacking around the architecture here.
```

Comment:
The error handling in these components didn't work. Also just consider what the state machine for this component would look like if it was modeled... 😵